tests/protocol/pool/Pool.Supply.Rwa.t.sol

```diff
@@ -4,35 +4,23 @@ pragma solidity ^0.8.0;
 import 'forge-std/Test.sol';
 
 import {TestnetProcedures} from 'tests/utils/TestnetProcedures.sol';
-import {AccessControl} from 'src/contracts/dependencies/openzeppelin/contracts/AccessControl.sol';
 import {IAToken, IERC20} from 'src/contracts/interfaces/IAToken.sol';
 import {Errors} from 'src/contracts/protocol/libraries/helpers/Errors.sol';
 import {TestnetERC20} from 'src/contracts/mocks/testnet-helpers/TestnetERC20.sol';
 import {EIP712SigUtils} from 'tests/utils/EIP712SigUtils.sol';
-import {SupplyLogic} from 'src/contracts/protocol/libraries/logic/SupplyLogic.sol';
+import {IPool} from 'src/contracts/interfaces/IPool.sol';
 
 contract PoolSupplyRwaTests is TestnetProcedures {
-  address internal aUSDX;
   address internal aBuidl;
-  address aTokenTransferAdmin;
 
   function setUp() public {
     initTestEnvironment();
-
-    aTokenTransferAdmin = makeAddr('ATOKEN_TRANSFER_ADMIN_1');
-
-    (aUSDX, , ) = contracts.protocolDataProvider.getReserveTokensAddresses(tokenList.usdx);
     (aBuidl, , ) = contracts.protocolDataProvider.getReserveTokensAddresses(tokenList.buidl);
 
     vm.startPrank(poolAdmin);
     // authorize & mint BUIDL to alice
     buidl.authorize(alice, true);
     buidl.mint(alice, 100000e6);
-    // grant Transfer Role to the aToken Transfer Admin
-    AccessControl(aclManagerAddress).grantRole(
-      keccak256('ATOKEN_TRANSFER_ROLE'),
-      aTokenTransferAdmin
-    );
     // authorize aBUIDL contract to hold BUIDL
     buidl.authorize(aBuidl, true);
```

Comment:
we don't need this anymore, it's already in the setup

Reply:
[bcba376](https://github.com/aave/aave-v3-horizon/pull/3/commits/bcba376f38fe08cdd47d58a4058a4dd6d79da396)